src/styles/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat($column-amount, 2);
+  gap: 46px 48px;
```

Comment:
The gap property values should be defined using variables for consistency and maintainability, as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Consider creating variables for these gap sizes if they are reused throughout the project.